index.js

```diff
@@ -231,7 +244,7 @@ async function handleSensorRequest(currentLocationId) {
 
   if (location_start_time !== null && location_start_time !== undefined) {
     const start_time_sesh = new Date(location_start_time)
-    const now = new Date()
+    const now = new Date(await db.getCurrentTime())
```

Comment:
Don't forget the try-catch around the db function call to avoid unhandle promise rejection warnings.